test/powershell/Host/ConsoleHost.Tests.ps1

```diff
@@ -244,17 +244,17 @@ Describe "ConsoleHost unit tests" -tags "Feature" {
 '@
```

Comment:
@iSazonov, your last commit had 2 failures in `PowerShell-CI-windows`
PSModulePath changes interacting with other PowerShell processes.Does not duplicate the System32 module path in subprocesses
```powershell
The expression after '&' in a pipeline element produced an object that was not valid. It must result in a command name, a script block, or a CommandInfo object.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\CompatiblePSEditions.Module.Tests.ps1: line 470
470:         $sys32ModPathCount = & $pwsh -C {

```
PSModulePath changes interacting with other PowerShell processes.System32 module path prepended to PSModulePath.Allows PowerShell subprocesses to call core modules
```powershell
The expression after '&' in a pipeline element produced an object that was not valid. It must result in a command name, a script block, or a CommandInfo object.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Core\CompatiblePSEditions.Module.Tests.ps1: line 464
464:             $errors = & $pwsh -Command "Get-ChildItem" 2>&1 | Where-Object { $_ -is [System.Management.Automation.ErrorRecord] }

```